r/geoarrow/R/sf-compat.R

```diff
@@ -5,6 +5,59 @@ st_as_sfc.geoarrow_vctr <- function(x, ..., promote_multi = FALSE) {
   wk::wk_set_crs(sfc, wk::wk_crs(x))
 }
 
+st_as_sfc.ChunkedArray <- function(x, ..., promote_multi = FALSE) {
+  vctr <- as_geoarrow_vctr(x)
+  st_as_sfc.geoarrow_vctr(vctr, ..., promote_multi = promote_multi)
+}
+
+st_as_sfc.Array <- function(x, ..., promote_multi = FALSE) {
+  vctr <- as_geoarrow_vctr(x)
+  st_as_sfc.geoarrow_vctr(vctr, ..., promote_multi = promote_multi)
+}
+
+st_as_sf.ArrowTabular <- function(x, ..., promote_multi = FALSE) {
+  df <- tibble::as_tibble(x)
```

Comment:
Do we need a tibble?

```suggestion
  df <- as.data.frame(x)
```